test/methods.test.js

```diff
@@ -1,6 +1,109 @@
-describe("Test for hasVowel method in the String Prototype", () => {
-    it("should returns 'true' if the string contains vowels", () => {
+require('../method/methods')
+
+describe("Test for hasVowel() method in the String Prototype", () => {
+    it("should return 'true' if the string contains vowels", () => {
         var result = "Going home".hasVowel();
         expect(result).toBeTruthy();
     });
+});
+
+describe("Test for toUpper() method in String Prototype", () => {
+    it("should return the passed-in string with all characters in upper cases", () => {
+        var result = "Going 3467 home".toUpper();
```

Comment:
Always use single quotes for Javascript strings

Reply:
okay. it has been effected.